src/ansys/fluent/core/rpvars.py

```diff
@@ -48,13 +50,23 @@ def __call__(self, var: str = None, val: Any=None) -> Any:
             self._get_var(var) if var else self._get_vars()
         )
 
+    def allowed_values(self) -> List[str]:
+        if not _allowed_rpvars_values:
+            _allowed_rpvars_values.append(lispy.parse(self._eval_fn("(cx-send '(map car rp-variables))")))
```

Comment:
Instead of having a list within list here, you can do the following:
```
class RPVars:
    _allowed_values = None
    @property
    def allowed_values(self) -> List[str]:
        if not RPVars._allowed_values:
            RPVars._allowed_values =  lispy.parse(self._eval_fn("(cx-send '(map car rp-variables))"))
        return RPVars._allowed_values
    def _get_var(self, var: str):
        if var not in self.allowed_values:
            ...
```

Reply:
Done, Thank you